documentation/reference/tensor.html

```diff
@@ -28,41 +29,52 @@ <h2 id="tensor-type-spec">Tensor type spec</h2>
 Specifies the tensor type for a tensor.
 A tensor type contains a list of dimensions on the format:
 <pre>
-tensor(dimension-1,dimension-2,...,dimension-N)
+tensor&lt;type&gt;(dimension-1,dimension-2,...,dimension-N)
```

Comment:
I suggest "value-type" as just "type" might be confusing (as the whole string is also a "type").